lib/urldata.h

```diff
@@ -1417,85 +1423,123 @@ struct Curl_multi;    /* declared in multihandle.c */
  */
 enum dupstring {
   STRING_CERT,            /* client certificate file name */
-  STRING_CERT_PROXY,      /* client certificate file name */
   STRING_CERT_TYPE,       /* format for certificate (default: PEM)*/
+  STRING_KEY,             /* private key file name */
+  STRING_KEY_PASSWD,      /* plain text private key password */
+  STRING_KEY_TYPE,        /* format for private key (default: PEM) */
+  STRING_SSL_CAPATH,      /* CA directory name (doesn't work on windows) */
+  STRING_SSL_CAFILE,      /* certificate file to verify peer against */
+  STRING_SSL_PINNEDPUBLICKEY, /* public key file to verify peer against */
+  STRING_SSL_CIPHER_LIST, /* list of ciphers to use */
+  STRING_SSL_CIPHER13_LIST, /* list of TLS 1.3 ciphers to use */
+  STRING_SSL_CRLFILE,     /* crl file to check certificate */
+  STRING_SSL_ISSUERCERT, /* issuer cert file to check certificate */
+  STRING_SERVICE_NAME,    /* Service name */
+#ifndef CURL_DISABLE_PROXY
```

Comment:
This is in direct contradiction to the comment above the enum which says:

**This enumeration MUST not use conditional directives (#ifdefs)**

But: the comment also refers to a (misspelled) source code file that was removed in 821d108fc9adfb, which makes me suspect maybe this comment does not apply anymore?

@monnerat can you fill in some blanks maybe?

Reply:
It seems the comment is obsolete: the current test is now based on `CURLOPT_*`, not on `STRING_*` anymore.

Reply:
@MAntoniak can you perhaps update that comment to reflect reality?

Reply:
If I understand correctly just remove it?

Reply:
Yes!